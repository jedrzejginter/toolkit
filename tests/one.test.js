function strip(code) {
  let lines = code.split('\n').filter(Boolean);
  const spaces = lines[0].replace(/^(\s+).+/, '$1').length;

  return lines.map((line) => line.slice(spaces)).join('\n') + '\n';
}

test('one', () => {
  expect(
    strip(`
      function a() {
        const b = 1;
        return b;
      }
  `),
  ).toMatchSnapshot();
});
