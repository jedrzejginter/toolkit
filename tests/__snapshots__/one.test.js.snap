// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`one 1`] = `
"function a() {
  const b = 1;
  return b;
}

"
`;
